camelot/view/controls/formview.py

```diff
@@ -166,6 +166,15 @@ def _header_data_changed(self, orientation, first, last):
     def _layout_changed(self, index=None, start=None, end=None):
         widget_mapper = self.findChild(QtWidgets.QDataWidgetMapper, 'widget_mapper' )
         if widget_mapper is not None:
+            if (widget_mapper.mappedWidgetAt(0) is None) and (widget_mapper.model().columnCount()>0):
```

Comment:
The delayed creation of the form widgets seems to have some side effects : 

- the application of the form state seems to be very inconsistent, thus windows dont shuffle in the right place
- the opening of the form itself appears to be faster, maybe due to not activating the widget mapper before most of the information is available

to be investigated what to do with this change ... which is in principle not really needed, since a 'subclass' delegatemanager could be made for form view that does know the field attributes ...